demo/kserve/scripts/install/kserve-install.sh

```diff
@@ -201,51 +220,53 @@ bash -x ./scripts/generate-wildcard-certs.sh ${BASE_CERT_DIR} ${DOMAIN_NAME} ${C
-then
-  OPERATOR_LABEL="name=rhods-operator"
-  oc create ns ${TARGET_OPERATOR_NS} -oyaml --dry-run=client | oc apply -f-  
-  oc::wait::object::availability "oc get project ${TARGET_OPERATOR_NS} " 2 60
-fi
-oc create -f custom-manifests/opendatahub/${TARGET_OPERATOR}-operators-2.x.yaml
-
-wait_for_pods_ready "${OPERATOR_LABEL}" "${TARGET_OPERATOR_NS}"
-oc wait --for=condition=ready pod -l ${OPERATOR_LABEL} -n ${TARGET_OPERATOR_NS} --timeout=300s 
-
-# Example CUSTOM_MANIFESTS_URL ==> https://github.com/opendatahub-io/odh-manifests/tarball/master
-if [[ -n "${CUSTOM_MANIFESTS_URL+x}" ]]
-then
-  echo
-  echo "Added custom manifest url into default dscinitializations"
-  oc patch dscinitializations default -p="[{\"op\": \"add\", \"path\": \"/spec/manifestsUri\",\"value\": \"${CUSTOM_MANIFESTS_URL}\"}]" --type='json'
+if [[ ${DEPLOY_OPERATOR} == "true" ]]
+  then
+    # Deploy odh/rhods operator
+    echo
+    echo "[INFO] Deploy odh/rhods operator"
+    echo
+    # Create brew catalogsource
+    if [[ ${TARGET_OPERATOR} == "brew" ]];
+    then
+      echo
+      echo "[INFO] Create catalogsource for brew registry"
+      echo
+      sed "s/<%brew_tag%>/$BREW_TAG/g" custom-manifests/brew/catalogsource.yaml |oc apply -f -
+      wait_for_pods_ready "olm.catalogSource=rhods-catalog-dev" "openshift-marketplace"
+      oc wait --for=condition=ready pod -l olm.catalogSource=rhods-catalog-dev -n openshift-marketplace --timeout=60s  
+    fi
+    OPERATOR_LABEL="control-plane=controller-manager"
+    if [[ ${TARGET_OPERATOR_TYPE} == "rhods" ]];
+    then
+      OPERATOR_LABEL="name=rhods-operator"
+      oc create ns ${TARGET_OPERATOR_NS} -oyaml --dry-run=client | oc apply -f-  
+      oc::wait::object::availability "oc get project ${TARGET_OPERATOR_NS} " 2 60
+    fi
+    oc create -f custom-manifests/opendatahub/${TARGET_OPERATOR}-operators-2.x.yaml
+
+    wait_for_pods_ready "${OPERATOR_LABEL}" "${TARGET_OPERATOR_NS}"
+    oc wait --for=condition=ready pod -l ${OPERATOR_LABEL} -n ${TARGET_OPERATOR_NS} --timeout=300s 
+
+    # Example CUSTOM_MANIFESTS_URL ==> https://github.com/opendatahub-io/odh-manifests/tarball/master
+    if [[ -n "${CUSTOM_MANIFESTS_URL+x}" ]]
+    then
+      echo
+      echo "Added custom manifest url into default dscinitializations"
+      oc patch dscinitializations default -p="[{\"op\": \"add\", \"path\": \"/spec/manifestsUri\",\"value\": \"${CUSTOM_MANIFESTS_URL}\"}]" --type='json'
+    fi
+  else
+    printf "[INFO] Skip deployment odh/rhods operator"
 fi
 
 echo
 echo "[INFO] Deploy KServe"
 echo
-# ODH 1.9 use alpha api so this logic needed but from ODH 1.10, this logic must be deleted.
-#oc create -f custom-manifests/opendatahub/kserve-dsc.yaml
-if [[ ${TARGET_OPERATOR_TYPE} == "rhods" ]]; then
-  oc create -f custom-manifests/opendatahub/kserve-dsc.yaml
-else
-  oc create -f custom-manifests/opendatahub/kserve-dsc-v1alpha1.yaml
+dsc_exists=$(oc get datasciencecluster default --all-namespaces)
```

Comment:
Although odh operator was uninstalled, afaik the CR can remain. So we need to check if odh operator is running or not.

Reply:
the reason behind this change is that if RHODS/ODH deployment is disabled and kserve is not enabled in the cluster,  the `oc create` would not enable kserve. So, by using patch, we ensure to enable the component